### DISCUSSION TITLE
Heli crashes?

### DISCUSSION THREAD

**Discussion by BlackMistK** [TEHE-Studios/ExpandedHelicopterEvents#366] [Q&A] [answered]
anyway to tell if a heli has crashed? I assume that the pilots are a sign but im unsure

**Comment by Chuckleberry-Finn** [comment, marked as answer]
You will hear a loud crash, and if you're close enough the event marker will show a crashed helicopter.

**Reply by BlackMistK** [reply]
alright thank you
